waku/waku_api/rest/doc/relayapi_auto_content_topic.yaml

```diff
@@ -0,0 +1,25 @@
+# /relay/v1/auto/messages/{contentTopic}:  # Note the plural in messages
```

Comment:
Another reason for not having contentTopic as a URL param is since contentTopic format recommends having `/`, a user will have to URLencode the contentTopic before invoking the API. It is non-standard but if we include contentTopic in requestBody then this issue can be avoided.

While testing with postman for example with contentTopic `/toychat/1/huilong/proto`, i had to pass `%2Ftoychat%2F1%2Fhuilong%2Fproto` in order for this to work.

Reply:
We had exactly this conversation in nwaku. 
So far our conclusion was, that even though it requires url encoding, having GET request in content body is against rest convention and Http recommendation. But I could live with it having to do so.
Major issue was that the nim-presto implementation we use is - in align with the http spec - forbids having content body for a GET request.
So in order to do so we need to modify our HttpServer implementation.

I'm not sure how it is in your go impl. Does that allows it?

Reply:
In golang, the way we are implementing REST methods i don't see any limitation with this. 
Will test once to confirm though.

Reply:
It is even more extensive in store api. But as I see its only a problem if one would issue such and api call by hand, in all other programmatic case url encoding shall not be an issue.